containers/ci/generate-python-sbom.sh

```
@@ -0,0 +1,69 @@
+
+if ! command -v uv >/dev/null 2>&1; then
+  echo "uv is required to build the Python wheel" >&2
+  exit 1
+fi
+
+mkdir -p "$OUTPUT_DIR"
+
+echo "==> Building Python wheel"
+uv build --wheel
+
+wheel="$(ls -1 dist/*.whl 2>/dev/null | head -1 || true)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)